README.md

```diff
@@ -19,3 +19,21 @@ Dieses Konzept definiert [Free Open Source Software](https://de.wikipedia.org/wi
 ## Keine Herstellerabhängigkeiten
 
 ## Client-Betriebssysteme und Office-Anwendungen
+
+## Lizenzen
+
+"Open Source" Software bedeutet, dass der Quellcode frei zugänglich ist - der Code ist _einsehbar_ und _prüfbar_, wie bei einem Auto, dem man in den Motorraum sehen kann.
+
+Lizenzverträge regeln, inwiefern der Quellcode auch _nutzbar_ ist.
+Es gibt eine [Vielzahl unterschiedlicher Open Source Lizenzen](https://choosealicense.com/).
+In erster Linie unterscheiden sich diese in der Anforderung, ob [Bearbeitungen](https://en.wikipedia.org/wiki/Derivative_work) des Quellcodes fortan der selben Lizenz unterliegen müssen - der sogenannten "[Copyleft](https://de.wikipedia.org/wiki/Copyleft)" Klausel oder auch "Viralität" einer Lizenz.
+
+"Virale" Lizenzen (wie die [GPL](https://choosealicense.com/licenses/gpl-3.0/) und die [EUPL](https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12)) erfordern, dass Bearbeitungen derselben Lizenz unterliegen wie das ursprüngliche Werk.
+So müssen Veränderungen am Quellcode auch den ursprünglichen Autoren zugänglich gemacht werden.
+Unterschiedliche Copyleft-Lizenzen können zueinander inkompatibel sein, was die Nutzung solcher Software erschweren kann.
+
+Bei "freizügigen" Lizenzen (wie der [MIT](https://choosealicense.com/licenses/mit/)) entfällt dieser Zwang.
+Auch Firmen und Entitäten, die ihre Arbeit nicht ohne weiteres veröffentlichen können, können solche Software ohne Bedenken nutzen und für ihre Zwecke anpassen.
+
+  - Die Stadt München soll Open Source Software unabhängig von diesen Kategorien nutzen können. Copyleft- wie auch freizügige Lizenzen sind besser als proprietäre Nutzungsverträge.
+  - Stellt die Stadt München selbst Software her, sollte sie eine freizügige Lizenz (wie MIT) wählen.
```

Comment:
Das würde ich nicht festschreiben. Wenn man den FOSS Gedanken wirklich fördern will, kann es opportun sein, eine CopyLeft Lizenz zu wählen, um andere auch zu altruistischem Verhalten zu zwingen.

Reply:
Klar, die Diskussion wird es noch geben müssen. Ich persönlich bin für freiere Lizenzen mit weniger bürokratischem Aufwand. MIT und BSD ist im Aufwind. Wenn eine Firma die Software nutzt, schadet es der Software nicht, kann aber was nutzen. Wenn die Firma die Software wegen der Lizenz nicht nutzt, hilft es der Software sicher nicht.

Reply:
Und ich hab generell ein Problem mit "zwingen". Zwang find ich nicht gut.

Reply:
tl;dr: Irgendeine Lizenz sollte IMHO schon festgeschrieben sein, sonst fängt diese Diskussion ja bei jedem neuen Softwareprojekt von neuem an. Welche ist zu diskutieren.

Reply:
Guten Abend, Ihr Lieben! :) Die Frage nach den Lizenzen haben wir vor Kurzem bezüglich der Eigenentwicklungen der LHM schon mal ausführlich diskutiert, die in Zukunft im Regelfall Open Source sein sollen. Dort haben wir uns auf EUPL/MIT und diese Formulierung geeinigt: "Der entwickelte Quellcode wird öffentlich zugänglich gemacht und unter eine freizügige Open Source-Lizenz (z.B. EUPL, MIT) gestellt. Um externe Beiträge zu ermöglichen, sind Contribution Rules zu formulieren."

Reply:
Generell ist die Stoßrichtung dieses Textes hier eine übergreifende Open Source-Strategie für die LHM - also nicht nur, was konkret mit den Eigenentwicklungen und beauftragter Software geschehen soll, sondern auch unsere Vision und wo es hingehen soll.